app/validationFunctions.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+let R = require('ramda')
+
+module.exports = {
+  isEmptyOrHasSpace (input) {
```

Comment:
This syntax of dropping the lambda operator `=>` and colon is new to me. What's the point? I prefer the homogenous syntax, where you can move functions in and out of objects.